### PR TITLE
add cadvisor

### PR DESCRIPTION
**Pull request by eranra**

### CODE REVIEW

File: run.sh [hunk 1]
```diff
@@ -99,6 +100,21 @@ forward_ports() {
...
+    PROJECT_CADVISOR=$(oc get project | grep "$CADVISOR_NS")
+    qr_targets=""
+    if [ -n "$PROJECT_CADVISOR" ]; then
+      echo -e "\nSetup port-forwards to cadvisor pods"
+      for name in $($KUBECTL -n "$CADVISOR_NS" get pod -o name); do
+          echo -e "\nSetup port-forward 310$i:3100 to cadvisor pod: $name"
+          (
+              $KUBECTL -n "$CADVISOR_NS" port-forward "$name" 310$i:8080;
+          ) &
+          qr_targets="$qr_targets'localhost:310$i',"
```
Comment (periklis) [inline run.sh:113]: Let's use here an enumeration like `808x`.

**Review by periklis — CHANGES_REQUESTED** [review]

File: hack/deploy_cadvisor.sh [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+}
+
+create_project() {
+  oc new-project "$1" > /dev/null 2>&1
+  echo "Done"
+}
+
+get_cadvisor() {
+  rm -r -f $CADVISOR_FOLDER
+  git clone https://github.com/google/cadvisor.git $CADVISOR_FOLDER
```
[the remark this answers is not in the document]
Comment (eranra) [reply hack/deploy_cadvisor.sh:46]: done

File: config/prometheus/config.template [hunk 3]
```diff
@@ -19,3 +19,7 @@ scrape_configs:
     scrape_interval: 10s
     static_configs:
     - targets: [{{LOKI_QUERIER_TARGETS}}]
+  - job_name: 'loki-querier'
```
[the remark this answers is not in the document]
Comment (eranra) [reply config/prometheus/config.template:22]: done

File: hack/deploy_cadvisor.sh [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+    echo "exposing $pod"
+    oc expose "$pod"
+  done
+  services=$(oc get services -o name)
+  for service in $services; do
+    echo "exposing $service"
+    oc expose "$service"
+  done
+  oc get routes
+}
```
[the remark this answers is not in the document]
Comment (eranra) [reply hack/deploy_cadvisor.sh:67]: removed, done

File: run.sh (hunk 1 above)
Comment (eranra) [reply run.sh:113]: done

**Review by periklis — APPROVED** [review]

**Pull request merged**